get_results.py

```diff
@@ -501,6 +502,7 @@ def __init__(self, csv_file_path, output_style="full"):
             "xfer_size":    "Xfer Size",
             "block_size":   "Block Size",
             "cont_rf":      "Cont RF",
+            "ec_cell_size": "Cell Size",
```

Comment:
I'll also add this to the database table

Reply:
Sounds good, Thanks. But this work we might want for temporary and later we can use the most common size only.

Reply:
Sure. It's easy to just drop the column later if we don't need it